Error when installing Mainsail or Fluidd

### Linux Distribution

PRETTY_NAME="Raspbian GNU/Linux 11 (bullseye)"
NAME="Raspbian GNU/Linux"
VERSION_ID="11"
VERSION="11 (bullseye)"
VERSION_CODENAME=bullseye
ID=raspbian
ID_LIKE=debian
HOME_URL="http://www.raspbian.org/"
SUPPORT_URL="http://www.raspbian.org/RaspbianForums"
BUG_REPORT_URL="http://www.raspbian.org/RaspbianBugs"

### What happened


```
╔═══════════════════════════════════════════════════════╗
║ ~~~~~~~~~~~~~~~~~~~~~ [ KIAUH ] ~~~~~~~~~~~~~~~~~~~~~ ║
║        Klipper Installation And Update Helper         ║
║ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ ║
╚═══════════════════════════════════════════════════════╝
╔═══════════════════════════════════════════════════════╗
║ ~~~~~~~~~~~~~~~~~~~ [ Main Menu ] ~~~~~~~~~~~~~~~~~~~ ║
╟──────────────────┬────────────────────────────────────╢
║  0) [Log-Upload] │   Klipper: Installed: 1            ║
║                  │     Owner: Klipper3d               ║
║  1) [Install]    │      Repo: klipper                 ║
║  2) [Update]     ├────────────────────────────────────╢
║  3) [Remove]     │ Moonraker: Installed: 1            ║
║  4) [Advanced]   │     Owner: Arksine                 ║
║  5) [Backup]     │      Repo: moonraker               ║
║                  ├────────────────────────────────────╢
║  S) [Settings]   │        Mainsail: Not installed     ║
║                  │          Fluidd: Not installed     ║
║ Community:       │   Client-Config: Fluidd-Config     ║
║  E) [Extensions] │                                    ║
║                  │   KlipperScreen: Not installed     ║
║                  │       Crowsnest: Not installed     ║
╟──────────────────┼────────────────────────────────────╢
║  v6.0.0-alpha.6  │  Changelog: https://git.io/JnmlX   ║
╟──────────────────┴────────────────────────────────────╢
║                        Q) Quit                        ║
╚═══════════════════════════════════════════════════════╝
``` 
Select 1 (Install)
Select either 3 or 4 (Install Mainsail or Fluidd)

Get following error:

```
###### Perform action: 4
[ERROR] An unexpected error occured:
invalid literal for int() with base 10: '127.0.0.128126'
Traceback (most recent call last):
  File "/home/pi/kiauh/kiauh/core/menus/base_menu.py", line 188, in run
    selected_option.method(
  File "/home/pi/kiauh/kiauh/core/menus/install_menu.py", line 89, in install_fluidd
    client_setup.install_client(FluiddData())
  File "/home/pi/kiauh/kiauh/components/webui_client/client_setup.py", line 97, in install_client
    ports_in_use: List[int] = read_ports_from_nginx_configs()
  File "/home/pi/kiauh/kiauh/components/webui_client/client_utils.py", line 361, in read_ports_from_nginx_configs
    ports_to_ints_list = [int(port) for port in port_list]
  File "/home/pi/kiauh/kiauh/components/webui_client/client_utils.py", line 361, in <listcomp>
    ports_to_ints_list = [int(port) for port in port_list]
ValueError: invalid literal for int() with base 10: '127.0.0.128126'
``` 


### What did you expect to happen

I expected it to install the software

### How to reproduce

Using KIIAH install Klipper, then install Moonraker. Try to install either of the web interfaces.

### Additional information

This is on a Raspberry Pi 4B with 8GB RAM. Standard Raspbian install. As it's in a shed I have an external USB Wifi adapter (BrosTrend AX1800 USB WiFi 6 Linux Compatible WiFi Adapter) with their driver and the internal Wifi turned off.

I also have Octopi installed and working fine.

--- a/.gitignore
+++ b/.gitignore
@@ -1,6 +1,10 @@
 .idea
 .vscode
 .pytest_cache
+.jupyter
+*.ipynb
+*.ipynb_checkpoints
+*.tmp
 __pycache__
 .kiauh-env
 *.code-workspace

--- a/kiauh/components/webui_client/client_utils.py
+++ b/kiauh/components/webui_client/client_utils.py
@@ -353,10 +353,16 @@ def read_ports_from_nginx_configs() -> List[int]:
             lines = cfg.readlines()
 
         for line in lines:
-            line = line.replace("default_server", "")
-            line = re.sub(r"[;:\[\]]", "", line.strip())
-            if line.startswith("listen") and line.split()[-1] not in port_list:
-                port_list.append(line.split()[-1])
+            line = re.sub(
+                r"default_server|http://|https://|[;\[\]]",
+                "",
+                line.strip(),
+            )
+            if line.startswith("listen"):
+                if ":" not in line:
+                    port_list.append(line.split()[-1])
+                else:
+                    port_list.append(line.split(":")[-1])
 
     ports_to_ints_list = [int(port) for port in port_list]
     return sorted(ports_to_ints_list, key=lambda x: int(x))

